### ISSUE TITLE
Names for colors

### ISSUE THREAD

**Issue by lucast15** [Toinane/colorpicker#52]
Thank you so much for this program, I use it every day.

What about names for the colors?
When there are very much colors in big projects, it can be very tricky to find the right one. 
And maybe groups could be good too, but not necessary.

**Comment by Sant268** [comment]
I could help you with this. Can I?

**Comment by Toinane** [comment]
@Sant268 Yeah you can! But don't need to change the source code, I'll add that in the next update: an entire rewrite of the application. You can make just a document for now.

I don't really statuate on how I'll done this feature:
- A color = a name
- Or a range of color = a name ?

- Just English ?
- Or a multi language name for color
